plugins/inputs/x509_cert/x509_cert_test.go

```diff
@@ -223,6 +223,35 @@ func TestTags(t *testing.T) {
 		t.Errorf("Expected a valid Hex serial number but got %s", acc.TagValue("x509_cert", "serial_number"))
 	}
 	assert.Equal(t, big.NewInt(1), serialNumber)
+
+	// expect root/intermediate certs (more than one cert)
+	assert.Greater(t, acc.NMetrics(), uint64(1))
```

Comment:
Please use `require` instead of `assert` as we are trying to unify these usages. Same below.

Reply:
Thanks Sven, I tried to rebase to current master and changed tests to use `require` and `os.CreateTemp` instead of `ioutil`.